clients/js/genjs.go

```diff
@@ -445,7 +445,8 @@ const methodTmplStr = `
 
       const requestOptions = {
         method: "{{.Method}}",
-        uri: this.address + "{{.PathCode}}",
+		uri: this.address + "{{.PathCode}}",
+		gzip: true,
```

Comment:
nit leading whitespace looks different

Reply:
Hmm shoot yeah totally missed this - thanks for the catch!